lib/templates/simple_template.dart

```diff
@@ -1,28 +1,24 @@
+import 'dart:io';
+
 import 'template.dart';
 import 'package:process_run/process_run.dart';
 
 class SimpleTemplate extends Template {
   @override
   Future<void> apply(String projectDir, Variables variables) async {
-    await run(
-      'cp',
-      [
-        '$templatesFolder/simple_main.dart.template',
-        '$projectDir/lib/main.dart',
-      ],
+    await copyFile(
+      '$templatesFolder/simple_main.dart.template',
+      '$projectDir/lib/main.dart',
     );
-    await run(
-      'cp',
-      [
-        '$templatesFolder/simple_pubspec.yaml.template',
-        '$projectDir/pubspec.yaml',
-      ],
+    await copyFile(
+      '$templatesFolder/simple_pubspec.yaml.template',
+      '$projectDir/pubspec.yaml',
     );
     await run(
```

Comment:
`sed` is obviously not cross-platform, so maybe it might be better to read this file into memory and perform sed like operations on it using RegExp?

Reply:
`sed` is actually UNIX standard, so there should be no issues cross-platform. I can improve on a followup to be made more on the dart side but I feel a version as-is is already a good launch

Reply:
What about windoze? I know it's hard to believe people still use that...

Reply:
if someone uses that thing they can send a pr :P